fakeserial.py

```diff
@@ -1,9 +1,20 @@
-from ctypes import Structure, c_int
+from ctypes import Structure, c_int, c_float
 import random
 import time
 
 class FakeDataStruct(Structure):
-    _fields_ = [('data1', c_int)]
+    _fields_ = [('yaw', c_float),
+                    ('pitch', c_float),
+                    ('roll', c_float),
+                    ('acceleration', c_float),
+                    ('velocity', c_float),
+                    ('rpm', c_int), 
+                    ('position', c_int),
+                    ('temperature_inside', c_float), 
+                    ('temperature_outside', c_float), 
+                    ('temperature_electronics', c_float),
+                    ('time_remaining', c_float),   
```

Comment:
Haha, true, unless we'd want to display as 1.5 mins instead of 1m30s :P. Anyway, I didn't pay much attention to the types here because they were just for testing.
Do you know what values we'd get for time? The number of seconds, and then we display it as minutes and seconds?